distributed/tests/test_core.py

```diff
@@ -194,6 +189,15 @@ async def test_server_listen():
     """
     Test various Server.listen() arguments and their effect.
     """
+    import socket
+
+    try:
+        EXTERNAL_IP4 = get_ip()
+        if has_ipv6():
+            EXTERNAL_IP6 = get_ipv6()
+    except socket.gaierror:
+        raise pytest.skip(reason="no network access")
```

Comment:
[`pytest.skip`](https://docs.pytest.org/en/latest/reference.html#pytest-skip) does not have a `reason` kwarg. Just use `raise pytest.skip("no network access")` 

```
[  346s]     @pytest.mark.asyncio
[  346s]     async def test_server_listen():
[  346s]         """
[  346s]         Test various Server.listen() arguments and their effect.
[  346s]         """
[  346s]         import socket
[  346s]     
[  346s]         try:
[  346s]             EXTERNAL_IP4 = get_ip()
[  346s]             if has_ipv6():
[  346s]                 EXTERNAL_IP6 = get_ipv6()
[  346s]         except socket.gaierror:
[  346s] >           raise pytest.skip(reason="no network access")
[  346s] E           TypeError: skip() got an unexpected keyword argument 'reason'
[  346s] 
[  346s] distributed/tests/test_core.py:199: TypeError

```

Reply:
Thanks @bnavigator! See https://github.com/dask/distributed/pull/4467